test/distributed/test_c10d.py

```diff
@@ -355,54 +353,42 @@ def test_compare_set(self):
         self.assertEqual(b"new_value0", new_value_result)
         self.assertEqual(b"new_value0", store.get("key0"))
 
-    def _create_client(self, addr, port, world_size):
+    def _create_client(self, addr, port, world_size, messages):
         try:
             client_store = dist.TCPStore(addr, port, world_size, timeout=timedelta(seconds=10))
             self.assertEqual(b"value", client_store.get("key"))
```

Comment:
I don't see any assertions for `set` and `compare_set`. Do we call them to verify that we don't get any `RuntimeError`? If that is the case, I recommend adding a quick note here.